lib/turn.rb

```diff
@@ -1,2 +1,40 @@
 class Turn
+  attr_reader :player,
+              :round
+
+  def initialize(player, round)
+    @player = player
+    @round = [0..10]
```

Comment:
i think we actually want there to be an unlimited amount of rounds

Reply:
Gotcha, I was thinking we could use the rounds to help signify if the player has lost or not.